src/main/kotlin/step3/domain/car/Car.kt

```diff
@@ -0,0 +1,14 @@
+package step3.domain.car
+
+class Car(val id: Int, private val carMovePointDecider: CarMovePointDecider) {
+
+    private var position: Int = 0
+
+    fun move() {
+        position += carMovePointDecider.getMovePoint()
+    }
+
+    fun getCarStatus(): CarStatus = CarStatus(id, position)
```

Comment:
자동차의 상태를 나타낸다면 함수보단 프로퍼티로 표현해주는 방식은 어떨까요?
```suggestion
    val status: CarStatus get() = CarStatus(id, position)
```